src/interfaces.ts

```diff
@@ -330,18 +330,18 @@ export interface IConversationDetails {
     description?: string;
     roles: IConversationRoles;
     isPublic: boolean;
-    participants?: IConversationParticipant[];
 }
 
 /**
  * Conversation details interface
  */
 export interface IConversationDetails2 extends IConversationDetails {
```

Comment:
Yes would definately cause a TS consumer a breaking change (so would have to go and update the sample app for instance)